usr/share/rear/verify/GALAXY11/default/410_ping_commcell.sh

```diff
@@ -0,0 +1,14 @@
+# ping Commvault server (CommServe)
+
+is_true "$PING" || return 0
+
+local commvault_instance_properties=/etc/CommVaultRegistry/Galaxy/Instance001/CommServe/.properties
+
+read junk CSHOSTNAME junk2 < <(
+	grep CSHOSTNAME $commvault_instance_properties
+)
+if test "$CSHOSTNAME" ; then
+	ping -c 5 -q "$CSHOSTNAME" >&2 || Error "Backup server [$CSHOSTNAME] not reachable via ping !"
```

Comment:
Only a side note because I spotted it (feel free to ignore).
I tink we have several such kind of code places.

In general a hardcoded error exit during "rear recover"
(after the disk layout was already successfully recreated)
is technically right but it is not very user friendly, cf.
https://github.com/rear/rear/issues/2695

It would be better to have a user dialog here so that
the user could use the ReaR shell to do "the right things"
to make his backup server accessible (if possible for him)
or he can deliberately abort "rear recover".

Reply:
Hmm, yes I see your point. And no, I think during `verify` it is safe to bail out via Error if the server is not reachable as the following restore for sure would not work.